cve_bin_tool/version_scanner.py

```diff
@@ -236,6 +236,11 @@ def run_java_checker(self, filename: str) -> Iterator[ScanInfo]:
                 version = root.find(schema + "version").text
             if version is None and parent is not None:
                 version = parent.find(schema + "version").text
+
+            # If all else fails, set version to UNKNOWN
```

Comment:
@Alienmaster pointed out that saying "else" here can make it confusing when there is no actual code `else` involved.  English is confusing as always.  I'm going to go ahead and try to change the comment to something clearer.